openstudiocore/src/model/ThermalZone.cpp

```diff
@@ -1998,18 +1998,13 @@ namespace detail {
   {
     boost::optional<AirLoopHVACSupplyPlenum> result;
 
-    std::vector<AirLoopHVACSupplyPlenum> plenums = model().getModelObjects<AirLoopHVACSupplyPlenum>();
+    std::vector<AirLoopHVACSupplyPlenum> plenums = getObject<ModelObject>().getModelObjectSources<AirLoopHVACSupplyPlenum>();
```

Comment:
Either one should be faster, we could experiment or just pick one, I don't have a big preference